dartagnan/src/main/antlr4/SpirvLexer.g4

```diff
@@ -1677,9 +1677,10 @@ Op : 'Op';
 Equals : '=';
 Id : '%' [A-Za-z0-9_]+;
 LiteralFloat : [+-]? [0-9]+ '.' [0-9]*;
-LiteralInteger : '-'? [0-9]+;
 LiteralUnsignedInteger : [0-9]+;
+LiteralInteger : '-'? [0-9]+;
 LiteralString : '"' ~[\n"]* '"';
+HeaderStart : ';' [ \t]* [=]+ [ \t]* [\n\r] -> pushMode(ModeHeader);
```

Comment:
I also thought about this. In principle one many `forall` assertions (this is similar to writing `assert` in a C program), but I don't know what the interpretation would be if one has several `exists` or mixed types. 

@natgavrilenko is there any particular reason why you wanted to add several outputs too?

Reply:
With multiple assertions, I think one can only interpret them as independent, i.e., you can do one SMT-query/verification run per assertion.
It is then possible to group same-type assertions (i.e. `forall/~exists` and `~forall/exists`) and do one query per group, resulting in at most two queries.

Reply:
In principle, we can have multiple assertions of different types, but it is not implemented in dartagnan and I don't think that we really need it now. 

The main use case that I have in my mind is about splitting a long assertion into multiple assertions of the same type in different lines for readability. I think it will be quite common that we want to verify some "forall" for each element of a long output array.

Reply:
I added support for only `forall/~exists` group assertions.